### PR TITLE
Settings page

### PR DESCRIPTION
**Pull request by isabellelii**
#### Description
Changes proposed in this pull request:

* Added forms to edit name, age, gender, email and password 
* Added profile picture and edit icon

#### What I have learned working on this feature: 
* How to add icons and avatars on ionic

#### Screenshots:
skarmavbild 2018-04-06 kl 10 33 41
skarmavbild 2018-04-06 kl 10 33 55

### CODE REVIEW

**Comment by NellaMori** [comment]
It looks cool, can we also include the option to change the language you want to learn and native perhaps? In case you select the wrong one when signing up.

**Review by Fleurtam — CHANGES_REQUESTED** [review]
looks nice!
small typos, in line 17 and 18 should be language instead of languange,
line 25 should be finnish with double n

File: src/pages/edit/edit.html [hunk 1]
```diff
@@ -1,7 +1,74 @@
...
+    <ion-item>
+      <ion-label>Languange I want to learn</ion-label>
+      <ion-select [(ngModel)]="languange">
+        <ion-option value="italian">Italian</ion-option>
+        <ion-option value="swedish">Swedish</ion-option>
+        <ion-option value="german">German</ion-option>
+        <ion-option value="french">French</ion-option>
+        <ion-option value="english">English</ion-option>
+        <ion-option value="spanish">Spanish</ion-option>
+        <ion-option value="finish">Finish</ion-option>
```
Comment (Fleurtam) [inline src/pages/edit/edit.html]: finish should be finnish

File: src/pages/edit/edit.html [hunk 2]
```diff
@@ -1,7 +1,74 @@
...
+      <ion-select [(ngModel)]="gaming">
+        <ion-option value="female">Female</ion-option>
+        <ion-option value="male">Male</ion-option>
+      </ion-select>
+    </ion-item>
+  </ion-list>
+  <ion-list>
+    <ion-item>
+      <ion-label>Languange I want to learn</ion-label>
+      <ion-select [(ngModel)]="languange">
```
Comment (Fleurtam) [inline src/pages/edit/edit.html]: languange should be language in line 62 and 63

**Review by Fleurtam — APPROVED** [review]
Nice!

**Pull request merged**